### DISCUSSION TITLE
Avoid Parallel Dest-Drive writing (4:0)

### DISCUSSION THREAD

**Discussion by wrzsef** [ericaltendorf/plotman#764] [Ideas]
Hi Team! Love this project. Great work so  far! 👍

As many users do, I also write dest on an external USB drive. Works fine, as long as not 2 or more jobs write their final plot simultaneously to the USB-HDD. If this happens, 4:0 takes extremely long, pushing the following jobs closer to and eventually into 4:0 and the whole process ends up in a mess. I know, with staggering time like 50min or so this should be avoided. However, there are still circumstances where two or more jobs enter phase 4:0 (btw: I believe it is 4:0 only which writes on dest, but I'm not 100% sure) -> if I see this happening, I manually use `plotman suspend` to stop a job and resume it as the other job has finished. 

Is there any way to tell plotman to **automatically suspend jobs** if two or more are doing 4:0 (or doing dest writing) to have only on job writing (similar to the archive feature)?

**Comment by BrunoCampana** [comment]
Thanks for clarify this, I had the exactly same issue and now I know how to proceed.
